ember-cli-build.js

```diff
@@ -1,13 +1,14 @@
+/*jshint node:true*/
 /* global require, module */
-var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
-module.exports = function(/*defaults*/) {
-  var app = new EmberApp({
-    snippetSearchPaths: ['tests/dummy/app']
```

Comment:
Is deletion of this OK for some reason? I believe it exists for the dummy app code snippet rendering.